### PR TITLE
Add support for lunches and EduPage's CDN.

### PR DESCRIPTION
**Pull request by DasMatus**
Working on some structs to allow working this with lunches. Everything regarding lunch [from the Python library](https://github.com/ivanhrabcak/edupage-api) is **finally** implemented.

### CODE REVIEW

**Comment by DasMatus** [comment]
The big number of changed files is caused by `cargo fmt`, since I was reformatting the code inside of `src/types/lunches.rs`.

File: src/lib.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #![doc = "If you don't know where to start, you can look at all the available methods in the [`edupage::Edupage`] struct documentation."]
 #![doc = include_str!("../README.md")]
-
+#![warn(clippy::all, clippy::pedantic)]
```
Comment (ivanhrabcak) [inline src/lib.rs:3]: Please remove this line. A workflow will be added that will run clippy and `cargo fmt` on PRs.

File: src/types/lunches.rs [hunk 2]
```diff
@@ -54,51 +73,185 @@ impl<Menu: Clone> Lunch<Menu> {
     pub fn sign_off(self, epage: Edupage) {
```
Comment (ivanhrabcak) [inline src/types/lunches.rs:73]: Please do not to use abbreviations

**Review by ivanhrabcak — COMMENTED** [review]
Thank you!

File: src/types/lunches.rs [hunk 3]
```diff
@@ -1,33 +1,51 @@
...
+    quality_average: Option<f32>,
+    quantity_average: Option<f32>,
+    quality_ratings: Option<u32>,
+    quantity_ratings: Option<u32>,
+}
+
+pub struct Lunches {
+    edupage: Edupage,
+}
+impl Lunch {
```
[the remark this answers is not in the document]
Comment (DasMatus) [reply src/types/lunches.rs:44]: Alright, will define the traits where they belong.

File: src/types/lunches.rs [hunk 4]
```diff
@@ -44,6 +62,7 @@ impl<Menu: Clone> Lunch<Menu> {
             .build()
             .unwrap();
     }
+    /// Choose the food
     pub fn choose(self, epage: Edupage, number: i32) {
```
[the remark this answers is not in the document]
Comment (DasMatus) [reply src/types/lunches.rs:66]: Will remove it, my fault.

File: src/lib.rs (hunk 1 above)
Comment (DasMatus) [reply src/lib.rs:3]: This is a leftover. Sorry for that.

Comment (DasMatus) [reply src/lib.rs:3]: Already resolved.

File: src/types/lunches.rs (hunk 2 above)
Comment (DasMatus) [reply src/types/lunches.rs:73]: OK. Fixed on next push.

**Comment by DasMatus** [comment]
Closing this PR because it has some problems and it has got too ambitious.

**Pull request closed without merging**